### PR TITLE
[mozart] Remove deprecated DisplayMetrics from ViewProperties.

### PR DESCRIPTION
**Pull request by mikejurka**

### CODE REVIEW

File: content_handler/runtime_holder.cc [hunk 1]
```diff
@@ -770,23 +770,17 @@ void RuntimeHolder::OnPropertiesChanged(
...
     viewport_metrics_.physical_padding_right =
-        layout->inset->right * pixel_ratio;
+        layout->inset->right;
     viewport_metrics_.physical_padding_bottom =
-        layout->inset->bottom * pixel_ratio;
-    viewport_metrics_.physical_padding_left = layout->inset->left * pixel_ratio;
-    viewport_metrics_.device_pixel_ratio = pixel_ratio;
+        layout->inset->bottom;
+    viewport_metrics_.physical_padding_left = layout->inset->left;
+    viewport_metrics_.device_pixel_ratio = 1.f;
```
Comment (abarth) [inline content_handler/runtime_holder.cc]: Where does the device pixel ratio come from now?

Comment (mikejurka) [reply content_handler/runtime_holder.cc]: SessionConnection::OnSessionEvents delivers updated metrics events, which I believe are the ones that end up actually being used. Still testing this change, I'll comment when that's done.

Comment (abarth) [reply content_handler/runtime_holder.cc]: If this value isn't being used, then we should set it to some bogus value like 0.f

Comment (mikejurka) [reply content_handler/runtime_holder.cc]: Looks like we're getting two different device pixel ratios in two different places, I can't quite grok it unfortunately. Either we need to set this to 1.f or I'll revert my mozart change and revisit this tomorrow with Chinmay.

**Comment by Hixie** [comment]
Can this PR be closed?

**Pull request closed without merging**